### DISCUSSION TITLE
登录时禁止登录

### DISCUSSION THREAD

**Discussion by theFSO** [mamoe/mirai#792] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/9400940/103308079-8b4d4900-4a4c-11eb-9cdf-83840fa98bad.png)
有解决的方法么

**Comment by Karlatemp** [comment]
更新至最新

**Comment by Him188** [comment, marked as answer]
使用 2.0-M2 并详细阅读新文档 Bots 章节

**Reply by Him188** [reply]
https://github.com/mamoe/mirai/blob/dev/docs/Bots.md#2-%E7%99%BB%E5%BD%95
